src/accelerate/inference.py

```diff
@@ -101,6 +102,10 @@ def pippy_forward(forward, num_chunks, *args, **kwargs):
         output = forward()
     else:
         forward()
+    if send_output_to_cpu:
+        output = gather_object([output if output is not None else object()])
```

Comment:
I wonder if accelerate has a broadcast or a broadcast_object API? Just thinking that may fit the usage here?

Reply:
We do, but we can't broadcast when `None` is an input so `gather_object` gets the job done